artifacts_test.py

```diff
@@ -65,9 +65,8 @@ def tearDown(self) -> None:
         super().tearDown()
 
     @contextmanager
-    def logged_subtest(self, action: str,
-                       trace_id:  str | None = None, metadata: dict[str, Any] | None = None) -> Iterator[None]:
-        with self.actions_log.action_scope(action, self.node.name, trace_id, metadata):
+    def logged_subtest(self, action: str) -> Iterator[None]:
```

Comment:
Why not include the target and metadata in the message inside this function instead? It would remove a lot of changes and kept backwards compatibility

Reply:
basically adding curly braces, colons and additional 'target' is redundant. Also I tried to 'compress' it sometimes by changing e.g. `{"duration": "60m"}` to `for 60m` (appended).

Reply:
I am fine with removing metadata, those are hard to add so they make sense grammatically, but target seems reasonable
```
if target is not None:
  action = f"{action} on node {target}"
```

Reply:
usually that fits, but not 100% time.
Basically, I already did all the work, and I'm not happy about changing it back.
As to compatibility - I'm not aware any other use than manual or mcp server (which can have backward compatibility there)